test/unit/models/classes/task/migration/service/QueueMigrationServiceTest.php

```diff
@@ -0,0 +1,159 @@
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+
+namespace oat\tao\test\unit\models\classes\task\migration\service;
+
+
+use common_report_Report;
+use core_kernel_classes_Resource;
+use oat\generis\test\MockObject;
+use oat\generis\test\TestCase;
+use oat\tao\model\task\migration\MigrationConfig;
+use oat\tao\model\task\migration\ResultUnit;
+use oat\tao\model\task\migration\ResultUnitCollection;
+use oat\tao\model\task\migration\service\QueueMigrationService;
+use oat\tao\model\task\migration\service\ResultFilter;
+use oat\tao\model\task\migration\service\ResultSearcherInterface;
+use oat\tao\model\task\migration\service\ResultUnitProcessorInterface;
+use oat\tao\model\task\migration\service\SpawnMigrationConfigServiceInterface;
+use oat\tao\model\task\migration\service\StatementLastIdRetriever;
+
+class QueueMigrationServiceTest extends TestCase
+{
+    /** @var QueueMigrationService */
+    private $subject;
+
+    /** @var StatementLastIdRetriever|MockObject */
+    private $statementLastIdRetrieverMock;
+
+    /** @var ResultUnitProcessorInterface|MockObject */
+    private $resultUnitProcessorMock;
+
+    /** @var ResultSearcherInterface|MockObject */
+    private $resultSearcherMock;
+
+    /** @var common_report_Report|MockObject */
+    private $reportMock;
+
+    /** @var ResultFilter|MockObject */
+    private $resoultFilterMock;
+
+    /** @var SpawnMigrationConfigServiceInterface|MockObject */
+    private $spawnMigrationConfigServiceMock;
+
+    public function setUp(): void
+    {
+        $this->resultSearcherMock = $this->createMock(ResultSearcherInterface::class);
+        $this->resultUnitProcessorMock = $this->createMock(ResultUnitProcessorInterface::class);
+        $this->statementLastIdRetrieverMock = $this->createMock(StatementLastIdRetriever::class);
+        $this->reportMock = $this->createMock(common_report_Report::class);
+        $this->resoultFilterMock = $this->createMock(ResultFilter::class);
+        $this->spawnMigrationConfigServiceMock = $this->createMock(SpawnMigrationConfigServiceInterface::class);
+        $this->subject = new QueueMigrationService();
+        $this->subject->setServiceLocator($this->getServiceLocatorMock([
+            StatementLastIdRetriever::class => $this->statementLastIdRetrieverMock,
+            ResultUnitProcessorInterface::class => $this->resultUnitProcessorMock,
+        ]));
```

Comment:
```suggestion
        $this->subject->setServiceLocator(
            $this->getServiceLocatorMock(
                [
                    StatementLastIdRetriever::class => $this->statementLastIdRetrieverMock,
                    ResultUnitProcessorInterface::class => $this->resultUnitProcessorMock,
                ]
            )
        );
```

PSR-12 as mentioned before: https://www.php-fig.org/psr/psr-12/